xml/s4s_setup_cluster.xml

```diff
@@ -685,16 +693,12 @@
    <step>
     <para>
      Import, validate, and install the cluster unattendedly,
+     <remark>sknorr 2019-05-29: Do I need to switch to a different machine now?</remark>
```

Comment:
The process needs to be started on the machine that will be the Primary HANA, as a consequence, the other machine will become the secondary on the HANA System Replication configuration.

Reply:
As mentioned before, step 2 & 3 are executed on the primary HANA machine. I've added the "On the primary HANA machine..." on step 3, so it should be clear that this step is also executed on the primary.